test/e2e/firewall.go

```diff
@@ -128,7 +128,7 @@ var _ = framework.KubeDescribe("Firewall rule", func() {
 		Expect(len(nodeTags.Items)).Should(Equal(1))
 
 		By("Checking if e2e firewall rules are correct")
-		for _, expFw := range framework.GetE2eFirewalls(cloudConfig.MasterName, masterTags.Items[0], nodeTags.Items[0], cloudConfig.Network) {
+		for _, expFw := range framework.GetE2eFirewalls(cloudConfig.MasterName, cloudConfig.MasterTag, cloudConfig.NodeTag, cloudConfig.Network, cloudConfig.ClusterIPRange) {
```

Comment:
Why are you changing nodeTag and masterTag params here? Was it part if the original PR or just an artifact of doing a rebase?

Reply:
:joy: obviously I'm rebasing by ignoring head

Reply:
fixed